src/wallet/rpcwallet.cpp

```diff
@@ -363,11 +363,29 @@ UniValue listaddresses(const UniValue& params, bool fHelp)
             "        ]\n"
```

Comment:
(2 lines above) Sapling addresses start with `zs1` (for Mainnet) and are Bech32.